content/en/docs/setup/additional-setup/cni/index.md

```diff
@@ -1,64 +1,78 @@
 ---
-title: Install Istio with the Istio CNI plugin
-description: Install and use Istio with the Istio CNI plugin, allowing operators to deploy services with lower privilege.
+title: Install the Istio CNI component
+description: Install and use the Istio CNI component, allowing operators to deploy workloads with lower privilege.
```

Comment:
Is `operators` valid since it can be deployed by the `cni` Helm chart as well? I guess the Operator and istioctl use the Helm chart.